debian/bullseye/foreman-release/foreman.list

```diff
@@ -0,0 +1,3 @@
+deb http://deb.theforeman.org/ bullseye nightly
+deb http://deb.theforeman.org/ plugins nightly
```

Comment:
```suggestion
deb [signed-by=/usr/share/keyrings/theforeman-archive-keyring.asc] http://deb.theforeman.org/ bullseye nightly
deb [signed-by=/usr/share/keyrings/theforeman-archive-keyring.asc] http://deb.theforeman.org/ plugins nightly
```

(and `/usr/share/keyrings/theforeman-archive-keyring.asc` should be created based on the current signing key we have today)

Reply:
I have added the commit for this, can you check if its correct?